pkg/ebpf/c/common/buffer.h

```diff
@@ -448,6 +449,10 @@ statfunc int events_perf_submit(program_data_t *p, u32 id, long ret)
     p->event->context.eventid = id;
     p->event->context.retval = ret;
 
+    init_task_context(&p->event->context.task, p->event->task, p->config->options);
```

Comment:
So this has no chance of failure now, which is because some instances of error handling were removed. What's the rationale behind the decision?

Reply:
We had error handling in 2 places:
1. bpf_get_current_comm fails only when there is no task (https://elixir.bootlin.com/linux/latest/source/kernel/bpf/helpers.c#L261) - this will never happen or we would have failed earlier anyway
2 . reading uts name using bpf_probe_read_kernel_str shouldn't happen as well, and if it does we should continue anyway (no reason to drop the event in that case)

Considering the above + for simplicity + to save some instructions for all programs (since all call this function) - I removed this error handling